README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# My name is Alia Ai-Ming Tehzeeb Shahzad. 
+
+I'm a **third year** at the College.
```

Comment:
Use of bold and italics does a good job of emphasizing what the important info is